README.md

```diff
@@ -220,6 +220,22 @@ gesdinet_jwt_refresh_token:
     token_parameter_name: refreshToken
 ```
 
+### Return Expiration Timestamp
+
+If set to true, the expiration timestamp will be added to the response.
```

Comment:
Perhaps also note that the timestamp is a Unix timestamp so folks don't expect a formatted DateTime.

Reply:
√